src/element/Note.css

```diff
@@ -1,35 +1,37 @@
 .note {
-    margin-bottom: 10px;
-    border-radius: 10px;
+    margin-bottom: 24px;
+    border-radius: 16px;
     background-color: var(--note-bg);
-    padding: 10px 10px 8px 10px;
+    padding: 24px;
```

Comment:
Way too much padding for mobile, almost no space left 
![theme snort-social pages dev_p_npub107jk7htfv243u0x5ynn43scq9wrxtaasmrwwa8lfu2ydwag6cx2quqncxg(Samsung Galaxy S20 Ultra)](https://user-images.githubusercontent.com/1172179/213430192-0a1dbb1b-005c-44a7-afe2-a5ddc39870c2.png)